src/stories/MovieCard.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { defineProps } from 'vue';
+
+interface Movie {
+    name: string;
+    genre: string;
+    year: number;
+    rating: number;
+    movieLength: number;
+    description: string;
+    poster: string;
+}
+defineProps<{ movie: Movie }>()
+</script>
+
+<template>
+    <div class="card">
+        <img 
+        class="card__img" :src="`${movie.poster}`" :alt="movie.name" />
```

Comment:
The "`" is not necessary:
```suggestion
        class="card__img" :src="movie.poster" :alt="movie.name" />
```

Reply:
Done